src/components/SideBar.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import { Divider, Drawer, IconButton, List, Typography, Link } from '@mui/material';
+import MenuIcon from '@mui/icons-material/Menu';
+import { Environment } from './Environment';
+import { Navigation } from './Navigation';
+
+function SideBar() {
+  const [openDrawer, setOpenDrawer] = useState(false);
+
+  return (
+    <div>
+      <Drawer PaperProps={{ sx: { width: '25%' } }} open={openDrawer} onClose={() => setOpenDrawer(false)}>
+        <List sx={{ marginTop: '20px' }}>
+          <Link
+            sx={{
+              color: 'black',
+              fontSize: '22px',
+              fontFamily: 'BlinkMacSystemFont',
+              paddingLeft: '16px'
+            }}
+            underline={'none'}
+            href="/"
+            onClick={() => setOpenDrawer(false)}
+          >
+            Home
+          </Link>
+        </List>
+        <Divider sx={{ paddingTop: '8px' }} orientation="horizontal" flexItem />
+        <Typography
+          sx={{
+            color: 'black',
```

Comment:
NIT (nit is short for nitpick, its a convention I use when I have a suggestion that isn't critical so can be ignored): Where is where we can leverage the theme provider to set the style for all `h5` variants.